web/form.py

```diff
@@ -69,7 +69,8 @@ class Meta:
 				'repository_link',
 				'website_production',
 				'website_test',
-				'project_owner',)
+				'project_owner',
+				'user_project',)
```

Comment:
I'd personally name this `project_members`, so it's more clear what the field refers to.